src/main/java/ch/njol/skript/structures/StructEvent.java

```diff
@@ -46,8 +46,10 @@ public class StructEvent extends Structure {
 	@SuppressWarnings("ConstantConditions")
 	public boolean init(Literal<?>[] args, int matchedPattern, ParseResult parseResult, EntryContainer entryContainer) {
 		String expr = parseResult.regexes.get(0).group();
+		EventData data = getParser().getData(EventData.class);
+		data.priority = null;
```

Comment:
it might be better to set it to null when we are done with parsing the SkriptEvent rather than before?

Reply:
I mean it might be good to set null on both sides, but I think it's more defensive to set it to null before.